troposphere/static/js/components/admin/ResourceRequest/Label.react.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+const Label = React.createClass({
+    render() {
+        let style = Object.assign({
+            margin: "7px 0px",
+        }, this.props.style);
+
+        let props = Object.assign({}, this.props, {
+            style
+        });
+
+        return (
+        <label { ...props }></label>
```

Comment:
Your example doesn't express what mine does. I'm trying to transparently apply all props. So if you pass an invalid attribute, react will warn, just like I want. This way `<Label>` is equivalent to `<label>`.